sketch.js

```diff
@@ -10,9 +10,18 @@ var decaySlider, decay;
 var sustainSlider, sustain;
 var releaseSlider, release;
 
+var keyWidth, keyHeight;
+var xTranslate, yTranslate;
+
 function setup() {
   createCanvas(1440, 800);
 
+  //Initializing some GUI element properties
```

Comment:
I know I didn't provide a style guide (I probably should...), but can you put a space after `//` for comments for consistency sake?